tests/test_font.py

```diff
@@ -46,6 +46,44 @@ def test_FontArguments_setCollectionIndex(fontarguments):
     fontarguments.setCollectionIndex(0)
 
 
```

Comment:
I think there needs to be a dozen more tests for each of the new properties/methods of the new classes added.